programming_foundations/lean_fridge_app_final_proj/sql_commands.py

```diff
@@ -0,0 +1,28 @@
+import sqlite3
+db = sqlite3.connect('lean_fridge.db')
+c = db.cursor()
+
+def itemTable():
+    c.execute("CREATE TABLE itemProperty(ID INT, name varchar(20), dates varchar(10), count INT, category varchar(20), note varchar(50))")
```

Comment:
for date, there are column type that are more suited. do you want to instead try using that?